ledger/state.go

```diff
@@ -921,10 +922,150 @@ func (ls *LedgerState) loadPParams() error {
 	if err != nil {
 		return err
 	}
-	ls.currentPParams = pparams
+	// Only overwrite currentPParams if we found something in the database
+	// or if currentPParams is nil
+	if pparams != nil || ls.currentPParams == nil {
+		ls.currentPParams = pparams
+	}
```

Comment:
I'm not sure we want to guard against `ls.currentPParams` already being populated. We have other cases where a load from the DB will overwrite existing cached data, such as `loadEpochs()`